### PR TITLE
[AIRFLOW-XXX] Remove duplicate information

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

The integration.rst file contains a lot of repeated information from the API reference documentation. Now the reference documentation is clearer and it makes no sense to repeat the information twice.

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by mik-laj** [comment]
PTAL @ashb

**Comment by mik-laj** [comment]
I rebased PR.  
New preview: http://tall-smile.surge.sh/integration.html
In addition, I corrected the names of the services to be consistent with the official naming and sorted alphabetically.

PTAL @kaxil  @ashb

File: docs/integration.rst [hunk 1]
```diff
@@ -100,163 +91,118 @@ Airflow connection of type `azure_data_lake` exists. Authorization can be done b
...
-For more instructions on using Amazon SageMaker in Airflow, please see `the SageMaker Python SDK README`_.
+* :py:mod:`airflow.hooks.S3_hook`
+* :py:mod:`airflow.operators.s3_file_transform_operator`
+* :py:mod:`airflow.contrib.operators.s3_list_operator`
+* :py:mod:`airflow.contrib.operators.s3_to_gcs_operator`
+* :py:class:`airflow.contrib.operators.gcp_transfer_operator.S3ToGoogleCloudStorageTransferOperator`
+* :py:mod:`airflow.operators.s3_to_hive_operator.S3ToHiveTransfer`
 
-.. _the SageMaker Python SDK README: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/workflow/README.rst
+Amazon Web Service Batch Service
```
Comment (ashb) [inline docs/integration.rst:186]: AWS vs "Amazon Web Service" (an it should be Service**s** I think if we expand it)

**Comment by OmerJog** [comment]
@mik-laj  ping :)

**Comment by mik-laj** [comment]
@OmerJog  Recently, I have not worked in the Airflow project. I am slowly returning to my tasks. This PR is not my main goal, because I already have another idea for this PR. I would like to clean up all classes in this Q - https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-21%3A+Changes+in+import+paths
Then this file will be much clearer.
In the next step, I think it's worth putting this document on a new website.
https://lists.apache.org/thread.html/982c7baa06742ad722f2baa0db53ad99aea6c26b14b7d6d4aa522677@%3Cdev.airflow.apache.org%3E

**Pull request closed without merging**